.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,34 @@
+If your issue is a **request for support in using Optuna**,
```

Comment:
For improving readability of the raw text (this is what users see immediately after pressing `New issue` button), I think it is better to rewrite the first sentence as follows (the generated HTML is identical as before):
```markdown
If your issue is a **request for support in using Optuna**, please post it on [Stack Overflow].
Support is also available on `#general` channel of [chainer slack] and [chainer-jp slack].

[Stack Overflow]: https://stackoverflow.com/questions/tagged/optuna
[chainer slack]: https://bit.ly/join-chainer-slack
[chainer-jp slack]: https://bit.ly/join-chainer-jp-slack
```

Reply:
Sounds good! I've fixed it.
Thank you for your suggestion.